### PR TITLE
[stable/gangway] Update to 3.2.0

### PR DESCRIPTION
**Pull request by gabibbo97**
Signed-off-by: Giacomo Longo <gabibbo97@gmail.com>

Also fixes compatibility with k8s 1.16

@rk295 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by gabibbo97** [comment]
Ping

File: stable/gangway/templates/ingress.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "gangway.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
+{{- if $.Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
```
Comment (rk295) [inline stable/gangway/templates/ingress.yaml:5]: 👍

**Review by rk295 — APPROVED** [review]
/lgtm

**Pull request merged**